src/bubbleSortRecursion.js

```diff
@@ -0,0 +1,13 @@
+export default function bubbleSort(numbers){
+  let sorted = true
```

Comment:
The standard implementation of `bubbleSort` just uses two nested for loops - I understand the implementation, but would recommend sticking to the standard, both for readability/maintainability by other developers, and to remove the additional boolean flag and `if`.